src/index.html

```diff
@@ -1,13 +1,424 @@
+                <p class="testimonial__phrase">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+              </div>
+            </div>
+          </div>
+          <!--  3:   -->
+          <!--  IMG:   -->
+          <div class="testimonials__item grid__item grid__item--9-12">
+            <div class="testimonial">
+              <div class="testimonial__img-box">
+                <img
+                  src="./images/photos/p3.png"
+                  alt="Max Henry Avatar"
+                  class="testimonial__img"
+                >
+              </div>
+              <!--  NAME:   -->
+              <div class="testimonial__content">
+                <a
+                  href="https://www.linkedin.com/in/maxghenry"
+                  class="testimonial__name"
+                >
+                  Max Henry
+                </a>
+                <span class="testimonial__role">Chief Design Officer</span>
+                <p class="testimonial__phrase">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- Contact us:  -->
+      <section class="page__section contacts">
+        <h2 id="contact" class="page__title">
+          Contact us
+        </h2>
+
+        <div class="grid grid--desktop">
+          <div class="grid__item grid__item--1-2">
+            <div class="contact">
+              <p class="contact__value-name">
+                Phone
+              </p>
+              <a href="tel:123455555555" class="contact__value">
+                +1 234 5555-55-55
+              </a>
+              <p class="contact__value-name">
+                Email
+              </p>
+              <a href="mailto:hello@miami.com" class="contact__value">
+                hello@miami.com
+              </a>
+            </div>
+          </div>
+          <div class="grid__item grid__item--4-5">
+            <div class="contact">
+              <p class="contact__value-name">
+                Address
+              </p>
+              <a
+                href="https://goo.gl/maps/fMQPSg1KytcrBzFdA"
+                class="contact__value"
+              >
+                400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401
+              </a>
+            </div>
+          </div>
+          <div class="grid__item grid__item--8-12">
+            <div class="contact">
+              <form
+                class="contact__form"
+                action="#"
+                id="form"
+                method="post"
+                onsubmit="return false"
+              >
+                <label for="name"></label>
+                <input
+                  id="name"
+                  class="contact__input"
+                  type="text"
+                  placeholder="Name"
+                  required
+                >
```

Comment:
if you have label and input together it's better to write input inside label and remove for attribute